### ISSUE TITLE
[URGENT] Milestone 3 Script/Pipeline error

### ISSUE THREAD

**Issue by ksedivyhaley** [UBC-MDS/DSCI_522_GROUP_312#57]
Makefile fails on script 3 (EDA) with error:

```
python scripts/eda_v2.py --train_path='data/train.csv' --out_folder_path='results/eda_charts/'
Traceback (most recent call last):
  File "scripts/eda_v2.py", line 139, in <module>
    main(train_path = opt["--train_path"], out_folder_path = opt["--out_folder_path"])
  File "scripts/eda_v2.py", line 35, in main
    train = pd.read_csv(train_path)
  File "C:\Users\7ks42\Anaconda3\lib\site-packages\pandas\io\parsers.py", line 685, in parser_f
    return _read(filepath_or_buffer, kwds)
  File "C:\Users\7ks42\Anaconda3\lib\site-packages\pandas\io\parsers.py", line 457, in _read
    parser = TextFileReader(fp_or_buf, **kwds)
  File "C:\Users\7ks42\Anaconda3\lib\site-packages\pandas\io\parsers.py", line 895, in __init__
    self._make_engine(self.engine)
  File "C:\Users\7ks42\Anaconda3\lib\site-packages\pandas\io\parsers.py", line 1135, in _make_engine
    self._engine = CParserWrapper(self.f, **self.options)
  File "C:\Users\7ks42\Anaconda3\lib\site-packages\pandas\io\parsers.py", line 1917, in __init__
    self._reader = parsers.TextReader(src, **kwds)
  File "pandas\_libs\parsers.pyx", line 382, in pandas._libs.parsers.TextReader.__cinit__
  File "pandas\_libs\parsers.pyx", line 689, in pandas._libs.parsers.TextReader._setup_parser_source
FileNotFoundError: [Errno 2] File b"'data/train.csv'" does not exist: b"'data/train.csv'"
```

Also and unrelated to the above error, having the Makefile call python3 instead of just python is causing problems on my machine. I don't think it *should* be causing problems so I'm checking to see if this is a weird configuration issue, and what best practice is.

**Comment by slbsolomon** [comment]
As discussed, I don't get this error. George and I are both running OS Catalina and Eithar is running a previous OS. Firas will try running on his machine (he also is running OS Catalina) and we will update from there.

**Comment by ksedivyhaley** [comment]
Update from Firas: the problem is that the eda command calls python3 while the others call python. Change this and it will run.

**Comment by slbsolomon** [comment]
Addressed in #62
